devel/api.md

```diff
@@ -881,6 +889,56 @@ Type: `integer`
 
 The Spanning Tree Protocol (STP) port cost for destinations via this port.
 
+# Team
+
+Besides basic interface properties, each Team interface state also contains a
+dictionary saved in key `Team.CONFIG_SUBTREE`. In addition, this dictionary
+contains a list of dictionaries saved in key `Team.PORT_SUBTREE` and a
+dictionary saved in key `Team.RUNNER_SUBTREE`.
+
+Example:
+
+```python
+{
+    Interface.KEY : [
+        Interface.NAME: "team0",
+        Interface.TYPE: InterfaceType.TEAM,
+        Interface.STATE: InterfaceState.UP,
+        Team.CONFIG_SUBTREE: {
+            Team.PORT_SUBTREE: [
+                {
+                    Team.Port.NAME: "eth1"
+                },
+                {
+                    Team.Port.NAME: "eth2"
+                }
+            ],
+            Team.RUNNER_SUBTREE: {
+                Team.Runner.NAME: Team.Runner.RunnerMode.LOAD_BALANCE
+            }
+        }
+    ]
+}
+```
+
+These values are based on the teamd JSON API. For further information please
+refer to [teamd manual page](https://www.mankier.com/5/teamd.conf)
```

Comment:
I would not suggest to use external third party website for manpage, you may just say ` man 5 tead.conf` or the upstream link
of that manpage: https://github.com/jpirko/libteam/blob/master/man/teamd.conf.5